mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -85,20 +138,24 @@ def delete_endpoint_record(
     ),
 ) -> Response:
     """
-    Clears endpoint record from KV by endpoint_id
+    Clears endpoint record from the DB based on endpoint_id.
+
+    :param project:       The name of the project.
+    :param endpoint_id:   The unique id of the model endpoint.
+    :param auth_info:     The auth info of the request.
+
+    :return: Starlette response object.
     """
+
     mlrun.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
         mlrun.api.schemas.AuthorizationResourceTypes.model_endpoint,
         project,
         endpoint_id,
         mlrun.api.schemas.AuthorizationAction.delete,
         auth_info,
     )
-    # get_access_key will validate the needed auth (which is used later) exists in the request
-    mlrun.api.crud.ModelEndpoints().get_access_key(auth_info)
```

Comment:
I don't know what was the original reason for this access key validation logic. I also talked to @theSaarco about it but anyway I have removed it from the APIs because it seems redundant... the returned value it not stored and anyway for the API we use the access key from the env (`os.environ.get("V3IO_ACCESS_KEY")`) and not from the `auth_info`.